src/Tracking/AdminSettings.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace Give\Tracking;
+
+use Give_Admin_Settings;
+
+/**
+ * Class AdminSettings
+ * @package Give\Tracking
+ *
+ * @since 2.10.0
+ */
+class AdminSettings {
+	const USAGE_TRACKING_OPTION_NAME = 'usage_tracking';
+
+	/**
+	 * Bootstrap
+	 *
+	 * @since 2.10.0
+	 */
+	public function boot() {
+		add_filter( 'give_get_settings_advanced', [ $this, 'addSettings' ] );
+	}
+
+	/**
+	 * Add admin settings.
+	 *
+	 * @since 2.10.0
+	 *
+	 * @param array $settings
+	 *
+	 * @return mixed
+	 */
+	public function addSettings( $settings ) {
+		if ( ! Give_Admin_Settings::is_setting_page( 'advanced', 'advanced-options' ) ) {
+			return $settings;
+		}
+
+		array_splice(
+			$settings,
+			-2,
+			0,
+			[
+				[
+					'name'    => __( 'Usage Tracking', 'give' ),
+					'label'   => esc_html__( 'Allow usage of GiveWP to be tracked.', 'give' ),
```

Comment:
Change the label to to "Anonymous Usage Tracking".

Reply:
fixed: https://github.com/impress-org/givewp/pull/5460#issuecomment-730881805